src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -345,16 +348,21 @@ def __init__(self, executable_path, default_args=None):
       raise fuchsia.errors.FuchsiaConfigError(
           ('FUCHSIA_PKEY_PATH, FUCHSIA_PORTNUM, or FUCHSIA_RESOURCES_DIR was '
            'not set'))
-    self.host = Host.from_dir(
-        os.path.join(fuchsia_resources_dir, self.FUCHSIA_BUILD_REL_PATH))
+    fuchsia_resources_dir_plus_build = os.path.join(fuchsia_resources_dir,
+                                                    self.FUCHSIA_BUILD_REL_PATH)
+    self.host = Host.from_dir(fuchsia_resources_dir_plus_build)
     self.device = Device(self.host, 'localhost', fuchsia_portnum)
-    # Fuchsia fuzzer names have the format {package_name}/{binary_name}.
-    # TODO(ochang): Properly handle fuzzers with '/' in the binary name.
-    package, target = environment.get_value('FUZZ_TARGET').split('/')
-    self.fuzzer = Fuzzer(self.device, package, target)
     self.device.set_ssh_option('StrictHostKeyChecking no')
     self.device.set_ssh_option('UserKnownHostsFile=/dev/null')
     self.device.set_ssh_identity(fuchsia_pkey_path)
+    # Fuchsia fuzzer names have the format {package_name}/{binary_name}.
+    # TODO(ochang): Properly handle fuzzers with '/' in the binary name.
+    package, target = environment.get_value('FUZZ_TARGET').split('/')
+    test_data_dir = os.path.join(fuchsia_resources_dir_plus_build, "test_data",
+                                 "fuzzing", package, target)
```

Comment:
I put `os.path.join('test_data', 'fuzzing')` into a constant, but i can't include that full path because we don't parse environment variables until the __init__ function.  Let me know if you want more moved into the class constants or if this looks good to you